doc/guides/kconfig/index.rst

```diff
@@ -326,6 +326,97 @@ error-prone, since it can be hard to spot that the same dependency is added
 twice.
 
 
+Why aren't other symbols getting updated when I change a symbol's value?
```

Comment:
Shouldn't this add "when I use menuconfig or guiconfig"?

Reply:
Maybe instead: ``Getting a dependent symbol value to update when using menuconfig``

Reply:
@carlescufi 
Felt like it made it a bit long.

@dbkinder 
Will work normally, e.g. when using `depends on`. It's only in this special case that it breaks.

Reply:
Changed the heading to `"Stuck" symbols in the menuconfig`. Is that better?

Could make it "menuconfig or guiconfig" (and do the same in other places that mention menuconfig), but I think it might be a bit redundant.

Reply:
@carlescufi ping

Reply:
I'd really prefer my original proposal. I think this is common enough that it warrants being very explicit about the actual problem description.

Reply:
Added another commit to the PR that mentions `guiconfig` wherever `menuconfig` is mentioned.